### ISSUE TITLE
Lithium produces worsening

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9897]
**Preferred term label:**
Lithium produces worsening

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Myxedema

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as

Exacerbated by lithium exposure
HP:0034956
